app/src/main/java/org/fossasia/openevent/app/utils/DateUtils.java

```diff
@@ -100,6 +100,8 @@ private static SimpleDateFormat getFormatter(@NonNull String format) {
 
     @NonNull
     private static Date getDate(@NonNull SimpleDateFormat formatter, @NonNull String isoDateString) throws ParseException {
+        if (isoDateString == null)
+            return new Date();
```

Comment:
please have a look at PR sent to your repo about attendee's `checkedIn` field correction in the layouts